tests/functional/u/use/use_maxsplit_arg.py

```diff
@@ -82,3 +82,12 @@ class Bar():
 a = "1,2,3".split('\n')[0]  # [use-maxsplit-arg]
 a = "1,2,3".split('split')[-1]  # [use-maxsplit-arg]
 a = "1,2,3".rsplit('rsplit')[0]  # [use-maxsplit-arg]
+
+# Test cases for false-positive reported in #4664
+# https://github.com/PyCQA/pylint/issues/4664
+source = 'A.B.C.D.E.F.G'
+other_iterable = range(5)
+i = 0
+for j in other_iterable:

```

Comment:
```suggestion
for j in range(5):
```
I would simplify it a bit